### DISCUSSION TITLE
How to delete a table row

### DISCUSSION THREAD

**Discussion by deanchin** [qax-os/excelize#1280] [Q&A] [answered]
I have a table and would like to delete rows in the table except the header row.  I tried using RemoveRow and it seemed to work partially by deleting the whole row in the sheet, but the table shows empty rows instead of the table rows actually being deleted.

Example below shows that I deleted rows 2 and 3.  The content was removed but the table still shows empty rows.
![image](https://user-images.githubusercontent.com/11054869/179135568-397e25ef-05a7-4fa2-9ed4-991d89960989.png)

Would like have it show as the table row deleted like the image below:
![image](https://user-images.githubusercontent.com/11054869/179135697-8924e316-6910-4a8d-bde7-87520a41a37b.png)

**Comment by xuri** [comment, marked as answer]
Sorry, this library doesn't support changing the range or deleting the existing table in the worksheet, and the table range also won't be adjusted after remove or inserting columns and rows currently. I will add this support later.

**Reply by deanchin** [reply]
Thank you for your quick response

**Reply by xuri** [reply]
I have added support for adjusting table range when removing and inserting column/row, please update the master branch code, this feature will be released in the next version, and I'll add a new function for deleting table later.
